tests/unit/test_testing_farm.py

```diff
@@ -1754,22 +1755,33 @@ def test_is_supported_architecture(target, use_internal_tf, supported):
 
 
 @pytest.mark.parametrize(
-    "comment,expected_identifier,expected_pr_arg",
+    "comment,expected_identifier,expected_labels,expected_pr_arg",
     [
         (
-            "/packit-dev test --identifier my-id-1 namespace-1/repo-1#33",
+            "/packit-dev test --identifier my-id-1 --labels label1,label2 namespace-1/repo-1#33",
             "my-id-1",
+            ["label1", "label2"],
             "namespace-1/repo-1#33",
         ),
         (
             "/packit-dev test namespace-2/repo-2#36 --identifier my-id-2",
             "my-id-2",
+            None,
+            "namespace-2/repo-2#36",
+        ),
+        (
+            "/packit-dev test namespace-2/repo-2#36 --labels label1 --identifier my-id-2",
```

Comment:
Uff, this took me a while to parse and understand what Packit should do here...;)

Reply:
But thanks for covering this usecase.